tempto-examples/docker/docker-compose.yml

```diff
@@ -30,6 +30,13 @@ services:
       POSTGRES_USER: blah
       POSTGRES_PASSWORD: blah
 
+  cassandra:
```

Comment:
How differently am I suppose to run an example? Tempto-examples are executed as a part of CI.